hvpy/api_groups/jpeg2000/get_jp2_image.py

```diff
@@ -0,0 +1,48 @@
+from typing import Optional
+from datetime import datetime
+
+from pydantic import validator
+
+from hvpy.io import HvpyParameters, OutputType
+
+
+class getJP2ImageInputParameters(HvpyParameters):
+    """
+    Handles the input parameters of the `getJP2Image API.
+
+    <https://api.helioviewer.org/docs/v2/api/api_groups/jpeg2000.html#getjp2image>`__.
+
+    Attributes
+    ----------
+    date : datetime
+        Desired date/time of the JP2 image.
+    sourceId : int
+        Unique image datasource identifier.
+    jpip : bool, optional
+        Returns a JPIP URI instead of the binary data of the image if set to `True`, defaults to `False`.
+    Json : bool, optional
+        Returns the JSON if set to `True`, defaults to `False`.
+    """
+
+    date: datetime
+    sourceId: int
+    jpip: Optional[bool] = False
+    Json: Optional[bool] = False
```

Comment:
```suggestion
    jpip: bool = False
    Json: bool = Field(False, alias="json")
```
You will need to add the import for `Field`. In addition,  we should be able to rename the field in the documentation string. 

You will have to change the tests if I recall as well.

Reply:
Using Alias, I found that `Json` field which should be replace with `json` (because of alias): Setting `Json: True` still remains `False` in the URL, this is the reason why the pytest fails (because instead of `JSON` response, it returns `RAW`).

Reply:
Is that not ok? The class still will print `Json` but you can set it by doing `json`.

Reply:
So sorry, I miss typed the message.
It doesn't  replace the alias, it is still `Json` instead of `json` in url.

Reply:
Yes, we still need the dict method we have for it to work.